docker-compose.yml

```diff
@@ -82,6 +82,16 @@ services:
       - ./scripts/wait-for-it.sh:/etc/lantern/wait-for-it.sh
     command: /etc/lantern/wait-for-it.sh lantern-mq:5672 -- ./main
 
+  shinydashboard:
+    build:
+      context: ./shinydashboard
+      args:
+        cert_dir: ./certs
```

Comment:
why are certs copied here and in the override file? expected them just to be needed in the override file

Reply:
Will remove